app-backend/db/src/main/scala/com/azavea/rf/database/TeamDao.scala

```diff
@@ -0,0 +1,83 @@
+package com.azavea.rf.database
+
+import com.azavea.rf.database.Implicits._
+import com.azavea.rf.datamodel._
+
+import doobie._, doobie.implicits._
+import doobie.postgres._, doobie.postgres.implicits._
+
+import cats._, cats.data._, cats.effect.IO, cats.implicits._
+
+import java.util.{Date, UUID}
+import java.sql.Timestamp
+
+import scala.concurrent.Future
+
+object TeamDao extends Dao[Team] {
+  val tableName = "teams"
+
+  val selectF = sql"""
+    SELECT
+      id, created_at, created_by, modified_at, modified_by, organization_id,
+      name, settings
+    FROM
+  """ ++ tableF
+
+  def create(
+    team: Team
+  ): ConnectionIO[Team] = {
+    (fr"INSERT INTO" ++ tableF ++ fr"""
+      (id, created_at, created_by, modified_at, modified_by, organization_id,
+      name, settings)
+    VALUES
+      (${team.id}, ${team.createdAt}, ${team.createdBy}, ${team.modifiedAt},
+      ${team.modifiedBy}, ${team.organizationId}, ${team.name}, ${team.settings})
+    """)
+    .update
+    .withUniqueGeneratedKeys[Team](
+      "id", "created_at", "created_by", "modified_at", "modified_by", "organization_id",
+      "name", "settings"
+    )
+  }
+
+  def createTeam(
+    teamCreate: Team.Create,
+    user: User
+  ): ConnectionIO[Team] = {
+    val team = teamCreate.toTeam(user)
+    this.create(team)
+  }
+
+  def getTeamById(teamID: UUID): ConnectionIO[Team] = {
```

Comment:
This is something I'd rather keep out of the DAO since it's a one-liner.

Reply:
If we think we could reduce code complexity with a method like this, we could generalize it so that it is available on every DAO.